incubator/drone/templates/configmap.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  server.debug.is.enabled: "true"
+
+  # You'll probably want to leave this alone, but you can point to external
+  # DB instances if you'd like: http://readme.drone.io/0.5/manage/database/
```

Comment:
The above link is dead and the  correct URL is http://readme.drone.io/admin/database-engines/